front-end/src/api/topics.js

```diff
@@ -82,13 +82,19 @@ export function putTopic(persistent, tenant, namespace, topic, data) {
   var url = `/${persistent}/${tenant}/${namespace}/${topic}`
   if (data > 0) {
     url += '/partitions'
+    return request({
```

Comment:
```suggestion
    return putTopicByPartition(persistent, tenant, namespace, topic, data);
```

Reply:
@nodece Thanks for pointing it out. Made the suggested changes.